content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -82,9 +68,11 @@ Use a YAML file to specify the Helm chart parameters for the [Datadog Agent char
 1. Configure the Datadog API and application key secrets:
    {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
 datadog:
+  site: {{< region-param key="dd_site" >}}
```

Comment:
Unfortunately, this won't render correctly because it's technically a shortcode nested inside another shortcode. To get this to work dynamically, we'd have to use a normal code fence for the example:

`````
```yaml
datadog:
site: {{< region-param key="dd_site" >}}
apiKeyExistingSecret: datadog-secret
appKeyExistingSecret: datadog-secret
```
`````

The downside is we won't be able to display the fancy filename like we currently have.

Reply:
Changed to 
```yaml
datadog:
  site: datadoghq.com
```